botogram/objects/callback_query.py

```diff
@@ -0,0 +1,33 @@
+"""
+    botogram.objects.callback_query
+    Representation of the callback query-related upstream API objects
+
+    Copyright (c) 2015 Pietro Albini <pietro@pietroalbini.io>
```

Comment:
I did nothing on that file: the copyright notice should be of the single file, not of the whole project, and you're the author of that file :)